flumine/markets/middleware.py

```diff
@@ -99,7 +100,44 @@ def _process_runner_removal(
                         extra=order.info,
                     )
                 else:
-                    if (
+                    # TODO: "Where an SP lay bet in a win market has a maximum odds limit specified,..."
+                    if isinstance(order.order_type, MarketOnCloseOrder):
```

Comment:
No problem with this but can we be consistant with the rest of the library and instead use:

```python
if order.order_type.ORDER_TYPE == OrderTypes.MARKET_ON_CLOSE:
```

Reply:
Don't we need to filter this on LAY bets only?

`
For SP back bets, Betfair will at no point amend either the stake or the odds requested by a customer, despite any non-runners or withdrawals. However customers who have selected the SP limit option can reduce the minimum SP odds they are prepared to accept on a selection.
`